src/main/java/Duke.java

```diff
@@ -1,10 +1,44 @@
+import command.Parser;
+import command.Storage;
+import command.Ui;
+import task.TaskList;
+
+import java.io.IOException;
+
+import static java.lang.Integer.parseInt;
```

Comment:
I'm not sure if this is actually needed as Java automatically imports the entire `java.lang.*` package -- maybe consider removing this and see if it still compiles?